package.json

```diff
@@ -31,9 +31,9 @@
   "bin": {
     "angularity": "./bin/cli.js"
   },
-  "engineStrict": true,
```

Comment:
enforce engine due to release task on mac only working on 

- node v0.10.36
- not node v0.10.26
- not node v0.12.0